sections/basics/attaching-additional-props.md

```diff
@@ -7,27 +7,26 @@ This way you can for example attach static props to an element, or pass a third-
 Here we render an `Input` component and attach some dynamic and static attributes to it:
 
 ```react
-const Input = styled.input.attrs(({ size }) => ({
+const Input = styled.input.attrs({
```

Comment:
This object-form attrs is actually deprecated, it'd be good to update it to function version instead:

```js
const Input = styled.input.attrs(props => ({ })`...`
```

Reply:
@probablyup 
I originally tried to write it that way, but the current version doesn't call `.attrs` as a function, it appears.

This site is currently using `"styled-components": "^4.0.0-beta.11", and from xmy testing it isnt until 4.2.0 where the function form works properly, but that breaks the app (.attrs is used in many places.)

The fix would be to update to 4.3.1 and update any `.attrs` app wide to use the function form right?  I would be happy to do the work and submit a PR.